trulens_eval/examples/experimental/cortex_finetuning_groundedness_ff.ipynb

```diff
@@ -0,0 +1,819 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    train_human_scores = [1, 2, 3, 4, 5, 4, 3, 2, 1, 5, 5, 5]
</pre><p>why are you using sample data here instead of the actual data?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
oops fixed now